jupyter_book/cli/pluggable.py

```diff
@@ -0,0 +1,54 @@
+"""Plugin aware click command Group."""
```

Comment:
Can we add documentation to the dev docs so that others knows how this works? Or, link to a `click` page describing this kind of thing?

Reply:
Yeh we can add it to the docs in a subsequent PR, along with the rest of the documentation